packages/orchestration/src/cosmos-api.ts

```diff
@@ -212,6 +212,13 @@ export type LiquidStakingMethods = {
   liquidStake: (amount: AmountArg) => Promise<void>;
 };
 
+export type LocalAccountMethods = {
+  /** deposit payment from zoe to the account */
```

Comment:
```suggestion
  /** deposit payment (from zoe, for example) to the account */
```

Zoe isn't the only source of payments.